server/util/capabilities/capabilities.go

```diff
@@ -43,18 +43,10 @@ func ToInt(caps []akpb.ApiKey_Capability) int32 {
 }
 
 func IsGranted(ctx context.Context, env environment.Env, cap akpb.ApiKey_Capability) (bool, error) {
```

Comment:
Seems like the behavior here is changing slightly in the case where auth is not set up -- does this cause any issue?

Reply:
It looks like this function is called in 3 places and all 3 have an additional check (e.g. AttachUserPrefix) that already perform the anon check so this seems like a safe/good change. Can you confirm this was the motivation for it?